docs/languages/polyglot.md

```diff
@@ -0,0 +1,100 @@
+
+- **Connecting to and querying from** Microsoft SQL Server databases and Kusto clusters
+
+- **Language server support** such as intellisense, autocompletion, and syntax highlighting _for each language_
+
+- **Variable sharing** between languages that ensures a continuous workflow within the notebook
+
+    *Variable sharing not available for HTML and Mermaid
+
+- **Variable Explorer** to check state of variables and which subkernel variables exist in
+
+- **Full portability between Jupyter/JupyterLab**
+
+To learn about all the features, visit the VS Code Marketplace [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode).
+
+## Prerequisites
+
+To use Polyglot Notebooks in VS Code you will need:
+
+- [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
+- [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+
+### Why do I need the .NET SDK?
+
+Polyglot Notebooks is powered by .NET Interactive which is a cutting edge and innovative engine that can run multiple languages and share variables between them. In Polyglot Notebooks, it behaves as the notebook kernel and is the reason the .NET 7 SDK is required at this time.
+
+## Getting Started
+
+1. To create your first polyglot notebook, open the command palette (_Ctrl+Shift+P on Windows, Cmd+Shift+P on iOS_) and select `Polyglot Notebooks: Create new blank
+notebook`, select `.ipynb`, and select the language you'd like to start with.
+
+    You can also create a notebook by creating a new file and giving it the extension `.ipynb`. In this case, make sure that .NET Interactive is selected as the kernel of the notebook in the top right.
+
+![](images/polyglot/kernel_picker.png)
+
+
+2. By default, added cells will be the same language as the cell above it. To change the language, click the language picker in the bottom right of the cell.
+
+![](images/polyglot/language_picker.png)
+
+3. Get coding!
+
+![](images/polyglot/polyglot_nb_example.png)
+
+## Working with Data
+
+Connect to and query Microsoft SQL Server databases and Kusto clusters.
+
+To get more detailed instructions on connecting to and working with data sources, visit our [Working with Data documentation](https://github.com/dotnet/interactive/blob/main/docs/working-with-data.md).
+
+
+![](images/polyglot/SQL_connection_example.png)
+
+## Language Server Support
+
+Each language in Polyglot Notebooks gets a first-class editing experience with language server support such as autocompletion, syntax highlighting, and signature help.
+
+
+## Variable Sharing & Variable Explorer
+
+Share variables between languages using the `#!share` magic command and check on the values of variables for each language using the variable explorer. Simply click the `Variables` icon in the global toolbar to open the variable explorer.
+
+![SQLJavaScript](images/polyglot/SQLJavaScript.mp4)
```

Comment:
This doesn't seem to be loading in the preview pane of this md file but perhaps that's expected - Just wanted to call that out.

Reply:
GitHub doesn't render .mp4 videos and they will need to be wrapped in a video element to display on the site. I can make that change.